### ISSUE TITLE
Feature Request: Add Powerful Build in Advancement

### ISSUE THREAD

**Issue by stunts7058** [foundryvtt/dnd5e#2743]
While creating races it would be nice to have Powerful Build be a trait that I can program into the race as I do with some of the other traits.

**Comment by Fyorl** [comment]
Reframing this as a request to allow Advancements to enable certain 'racial traits' on the character. Currently there's only Halfling Lucky and Powerful Build. Not sure Elven Accuracy really counts as it's a feat.

**Comment by stunts7058** [comment]
Yes I should have fleshed the request out better. Not only the racial features, but all of the options in there would be nice to be accessible from the advancement menus. Advantage on initiative, reliable talent, diamond soul, etc. Not sure if it would be possible to add in the changes to critical threshold or extra damage dice, but that would be nice as well as it would be something else I could remove from my list of effects.
